README.md

```diff
@@ -1,13 +1,8 @@
 This is the repository for **Lean 4**.
 
-We provide [nightly releases](https://github.com/leanprover/lean4-nightly/releases)
-and have just begun regular [stable point releases](https://github.com/leanprover/lean4/releases).
-
```

Comment:
Like on lean-lang.org, do not lead people away from the quickstart towards direct binary releases